app/styles/fonts.js

```diff
@@ -7,9 +9,14 @@ export const sizes = {
   huge: "48px"
 };
 
-export const weights = {
+export const fontWeights = {
   lighter: "200",
   light: "300",
   normal: "400",
   bold: "600"
 };
+
+export const Heading = styled("div")`
```

Comment:
This probably shouldn't be in fonts

Reply:
Fine for now, until we have a better place for it